### PR TITLE
fix "package" github action

### PR DESCRIPTION
**Pull request by mattmessinger**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -79,6 +80,9 @@ jobs:
           mv index.yaml.tmp index.yaml
           popd
 
+          # update the README
+          sed -i "s/version: [[:digit:]].[[:digit:]].[[:digit:]]/version: ${{ env.CHART_VERSION }}/" README.md
```
Comment (matth-boise) [inline .github/workflows/main.yml:84]: @mattmessinger does CHART_VERSION start with a `v` ?  e.g., `v1.0.8` ?

Comment (mattmessinger) [reply .github/workflows/main.yml:84]: code removes the "v" around line 39

Comment (matth-boise) [reply .github/workflows/main.yml:84]: indeed.  thx!

**Review by matth-boise — APPROVED** [review]
thanks!

**Pull request merged**